src/index.html

```diff
@@ -11,7 +11,129 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" >
+      <fieldset class="section">
+        <legend>Personal informatiom:
+        </legend>
+        <label class="input">Surname:
+          <input 
+            type="text" 
+            autocomplete="off" 
+            name="sureName"
+          >
+        </label><br>
```

Comment:
Why do you need <br> here? 
```suggestion
        </label>
```

Reply:
Привіт в прикладі зазначено що ці поля мають бути окремі